devU-client/src/components/pages/courseUpdatePage.tsx

```diff
@@ -67,9 +74,10 @@ const CourseUpdatePage = ({}) => {
     return (
     <PageWrapper>
         <h1>Course Detail Update</h1>
-        <TextField id='name' label='Course Name' onChange={handleChange}/>
-        <TextField id='number' label='Course Number' onChange={handleChange}/>
-        <TextField id='semester' label='Semester' onChange={handleChange}  placeholder='Ex. f2022, w2023, s2024, u2025'/>
+        <TextField id='name' label='Course Name' onChange={handleChange} className={invalidFields.get('name')}/>
+        <TextField id='number' label='Course Number' onChange={handleChange} className={invalidFields.get('number')}/>
+        <TextField id='semester' label='Semester' onChange={handleChange} placeholder='Ex. f2022, w2023, s2024, u2025'
+                   className={invalidFields.get('semester')}/>
         <DatePicker selected = {startDate} onChange={handleStartDateChange}/>
```

Comment:
this is because the section is only datePicker it does not have any style for me to change, Kevin's pr update it to have the label for the date, and at that time I will update to highligt the name